src/pages/MetricDetail.svelte

```diff
@@ -176,6 +177,21 @@
       <td>Version</td>
       <td>{metric.version || 0}</td>
     </tr>
+    <tr>
+      <td>BigQuery</td>
+      <td>
+        {#each metric.bigquery_names.stable_ping_table_names as name}
+          {#if name[0] !== 'events'}
+            <a
```

Comment:
I think it would be nice to have a URL like:

```html
In <a href="/apps/fenix/tables/activation">org_mozilla_fenix.activation</a> as <a href="/apps/fenix/tables/activation?search=metrics.string.activation_identifier">metrics.string.activation_identifier</a>
```

That way people can choose either to browse the full schema, or start off with a specific search. They can also just copy the BigQuery column name directly from the ping page, if that's what they want to do.